src/App.js

```diff
@@ -1,13 +1,22 @@
 import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
 import DisplayQuotes from './components/quote';
+import Navigation from './components/Navigation';
+import HomePage from './components/Home';
 
 function App() {
   return (
-    <div className="App">
-      <Calculator />
-      <DisplayQuotes />
-    </div>
+    <Router>
+      <div className="App">
+        <Navigation />
+        <Routes>
+          <Route path="/Home" element={<HomePage />} />
+          <Route path="/Calculator" element={<Calculator />} />
```

Comment:
- [ ] Good job so far but when when run the app it shows a blank page, The default page should be the home page and to fix it you just need to change the path to `/` instead of `/Home` 🙃 🙃 

<img width="973" alt="Screenshot 2023-08-30 at 22 11 36" src="https://github.com/Omwomo/math-magicians/assets/74501523/d024917e-f3f5-4df2-80e3-a0d716ff4396">

- [ ] As mentioned in the project requirements we should follow the given design. The calculator page has a litile to the left and the calculator to the right 😁 😁 

Your design

<img width="973" alt="Screenshot 2023-08-30 at 22 13 58" src="https://github.com/Omwomo/math-magicians/assets/74501523/143e3d89-8ae2-4ba2-87fe-7d3d3869cb96">

Given design

<img width="908" alt="Screenshot 2023-08-30 at 22 15 56" src="https://github.com/Omwomo/math-magicians/assets/74501523/4c0b3cb8-dc23-4317-baf6-11396abcf2be">

- [Optional] I think it would be better to have friendly and human content on the home page Instead of what you did in the project you can add that content in your README file instead of adding it to your home page. You add to your home page texts about mathematics... 😁 😁